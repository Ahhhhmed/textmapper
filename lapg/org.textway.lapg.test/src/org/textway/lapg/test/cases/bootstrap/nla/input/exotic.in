nop;
result = aaa->invoke();
result = {-> 4; }->invoke();
