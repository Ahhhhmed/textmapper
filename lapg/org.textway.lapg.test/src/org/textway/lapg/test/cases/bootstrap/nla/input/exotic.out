shift: Lnop (nop)
shift: ';' (;)
reduce to statement
reduce to statements
shift: identifier (result)
shift: '=' (=)
shift: identifier (aaa)
reduce to primary_expression
shift: '->' (->)
shift: Linvoke (invoke)
reduce to exotic_call_prefix
shift: '(' (()
shift: ')' ())
reduce to exotic_call
reduce to primary_expression
reduce to unary_expression
reduce to binary_op
reduce to conditional_expression
reduce to assignment_expression
reduce to expression
shift: ';' (;)
reduce to statement
reduce to statements
shift: identifier (result)
shift: '=' (=)
shift: '{' ({)
shift: '->' (->)
shift: icon (4)
reduce to primary_expression
reduce to unary_expression
reduce to binary_op
reduce to conditional_expression
reduce to assignment_expression
reduce to expression
shift: ';' (;)
reduce to statement
reduce to statements_noreset
shift: '}' (})
reduce to closure
reduce to closure_rule
reduce to primary_expression
shift: '->' (->)
shift: Linvoke (invoke)
reduce to exotic_call_prefix
shift: '(' (()
shift: ')' ())
reduce to exotic_call
reduce to primary_expression
reduce to unary_expression
reduce to binary_op
reduce to conditional_expression
reduce to assignment_expression
reduce to expression
shift: ';' (;)
reduce to statement
reduce to statements
reduce to input
shift: eoi ()
